scico/linop/radon_svmbir.py

```diff
@@ -39,13 +41,18 @@ def __init__(
         input_shape: Shape,
         angles: np.ndarray,
         num_channels: int,
+        is_masked: Optional[bool] = False,
     ):
         """
         Args:
             input_shape: Shape of the input array.
             angles: Array of projection angles in radians, should be
               increasing.
             num_channels: Number of pixels in the sinogram
+            is_masked: If True, valid region of the
+                image is set to a circular region prescribed in the
+                image array. Otherwise, the whole image array is valid
+                for projection and reconstruction.
```

Comment:
"prescribed in the image array" is a little opaque.

Reply:
How about:
 If True, valid region of the
image is set to the largest circle that is prescribed
within the image array.
Otherwise, the whole image array is valid
for projection and reconstruction.

Reply:
I suggest: If True, the valid region of the image is determined by a mask defined as the circle inscribed within the image boundary.  Otherwise ....

Is it clear from context that "image" refers to the reconstruction and not the sinogram?

Reply:
Made the changes. I added more context so that it is clear what 'image' refers to.